terraform/collector/main.tf

```diff
@@ -203,3 +203,58 @@ resource "aws_security_group_rule" "allow_security_hub_collector_to_execute_api"
   security_group_id        = var.execute_api_vpc_endpoint_security_group_id
   source_security_group_id = module.security_hub_collector_runner.task_security_group_id
 }
+
+resource "aws_sns_topic" "alarm" {
+  name              = "security-hub-collector-dev-alarm"
+  kms_master_key_id = module.sns_kms_key.id
+}
+
+module "sns_kms_key" {
+  source = "github.com/Enterprise-CMCS/mac-fc-shared//lib/terraform/sns_kms_key?ref=e762290"
+
+  alias = "security-hub-collector-dev-sns"
+
+  aws_event_source_principals = [{
+    type        = "Service"
+    identifiers = ["events.amazonaws.com"]
+  }]
+}
+
+resource "aws_sns_topic_subscription" "alarm" {
+  topic_arn = aws_sns_topic.alarm.arn
+  protocol  = "email"
+  endpoint  = "cms-macfc@corbalt.com"
+}
+
+resource "aws_sns_topic_policy" "alarm" {
+  arn    = aws_sns_topic.alarm.arn
+  policy = data.aws_iam_policy_document.alarm_topic.json
+}
+
+data "aws_iam_policy_document" "alarm_topic" {
+  statement {
+    sid    = "AllowTaskFailureAlertToAlarmTopic"
```

Comment:
Nits: Default effect is `Allow` and statement ID maybe not necessary in a policy with one statement.